### PR TITLE
Feature/ddbteam 826 symfony upgrade

### PR DESCRIPTION
**Pull request by turegjorup**

### CODE REVIEW

File: .php_cs.cache [hunk 1]
```diff
@@ -0,0 +1 @@
...ray":true,"yoda_style":true,"blank_line_after_opening_tag":true,"compact_nullable_typehint":true,"declare_equal_normalize":true,"lowercase_cast":true,"lowercase_static_reference":true,"new_with_braces":true,"no_blank_lines_after_class_opening":true,"no_leading_import_slash":true,"no_whitespace_in_blank_line":true,"ordered_class_elements":{"order":["use_trait"]},"return_type_declaration":true,"short_scalar_cast":true,"single_blank_line_before_namespace":true,"single_trait_insert_per_statement":true,"ternary_operator_spaces":true},"hashes":{"src\/Command\/ParseFeedCommand.php":2209856204,"src\/Command\/ParseUserClickedCommand.php":1833400797,"src\/Controller\/IndexController.php":372058111,"src\/Entity\/SearchFeed.php":2405036947,"src\/Entity\/UserClickedFeed.php":1798017311,"src\/EnvVarProcessor\/TraceIdEnvVarProcessor.php":3139574186,"src\/Kernel.php":3015131646,"src\/Logger\/TraceIdProcessor.php":4180914678,"src\/Repository\/SearchFeedRepository.php":3478131344,"src\/Repository\/UserClickedFeedRepository.php":2592708904,"src\/Service\/FileDownloaderService.php":4012457535,"src\/Service\/ParseSearchFeedService.php":2100178450,"src\/Service\/ParseUserClickedService.php":2053911934}}
```
Comment (cableman) [inline .php_cs.cache]: This cache file should not be in git!

File: config/packages/doctrine.yaml [hunk 2]
```diff
@@ -1,29 +1,42 @@
...
-        # configure these for your database server
-        driver: 'pdo_mysql'
-        server_version: '10.6.7-MariaDB'
-        charset: utf8mb4
-        default_table_options:
-            collate: utf8mb4_unicode_ci
-
+        # IMPORTANT: You MUST configure your server version,
+        # either here or in the DATABASE_URL env var (see .env file)
+        server_version: 'mariadb-10.6.7'
```
Comment (cableman) [inline config/packages/doctrine.yaml]: We should define this in connecting string to enable chances in helm charts without need to make new releases, if updating the database server.

File: src/Controller/IndexController.php [hunk 3]
```diff
@@ -56,15 +51,14 @@ public function index(Request $request): Response
...
-     * @return string
      *   Formatted size string
      */
     private function formatByteSize(int $bytes): string
     {
-        if ($bytes >= 1073741824) {
-            $bytes = number_format($bytes / 1073741824, 2).' GB';
-        } elseif ($bytes >= 1048576) {
-            $bytes = number_format($bytes / 1048576, 2).' MB';
+        if ($bytes >= 1_073_741_824) {
```
Comment (cableman) [inline src/Controller/IndexController.php:58]: What is this changes with the "_" in the numbers

**Review by cableman — CHANGES_REQUESTED** [review]

File: src/Controller/IndexController.php (hunk 3 above)
Comment (turegjorup) [reply src/Controller/IndexController.php:58]: https://wiki.php.net/rfc/numeric_literal_separator

**Review by cableman — APPROVED** [review]

**Pull request merged**